src/main/java/org/team340/robot/subsystems/GooseneckRollers.java

```diff
@@ -0,0 +1,70 @@
+package org.team340.robot.subsystems;
+
+import com.ctre.phoenix6.hardware.CANdi;
+import com.ctre.phoenix6.hardware.TalonFXS;
+import edu.wpi.first.wpilibj2.command.Command;
+
+import static edu.wpi.first.wpilibj2.command.Commands.*;
+
+import java.util.function.DoubleSupplier;
+import org.team340.lib.util.command.GRRSubsystem;
+import org.team340.robot.Constants.RobotMap;
+
+public class GooseneckRollers extends GRRSubsystem {
+
+    final double kIntakeSpeed = 0.0;
+    final double kScoreSpeed = 0.0;
+    final double kIndexingSpeed = 0.0;
+
+    final TalonFXS rollerMotor;
+    final CANdi beamBreak;
+
+    public GooseneckRollers() {
+        rollerMotor = new TalonFXS(RobotMap.kGooseneckRollersMotor);
+        beamBreak = new CANdi(RobotMap.kGooseneckRollersBeamBreak);
+    }
+
+    public void applySpeed(double speed) {
+        rollerMotor.set(speed);
+    }
+
+    public void stop() {
+        rollerMotor.stopMotor();
+    }
```

Comment:
We should avoid directly exposing access to actuators from subsystems, commands should fill this role as it provides an inherit mutex to hardware interactions, which is safer. These methods could just be removed, as their implementation is quite simple so code duplication is not a concern.